hedera-node/hedera-app-spi/src/test/java/com/hedera/node/app/spi/state/WritableKVStateBaseTest.java

```diff
@@ -261,12 +263,103 @@ void getForModifyAfterRemove() {
 
             // Commit should cause the value to be removed
             state.commit();
-            Mockito.verify(state, Mockito.never()).putIntoDataSource(Mockito.anyString(), Mockito.anyString());
-            Mockito.verify(state, Mockito.times(1)).removeFromDataSource(Mockito.anyString());
+            Mockito.verify(state, Mockito.never()).putIntoDataSource(anyString(), anyString());
+            Mockito.verify(state, Mockito.times(1)).removeFromDataSource(anyString());
             Mockito.verify(state, Mockito.times(1)).removeFromDataSource(A_KEY);
         }
     }
 
+    @Nested
+    @DisplayName("size")
+    final class SizeTest {
+        /**
+         * Gives size of backing store. It doesn't consider modifications and read keys maps.
+         * If a new key is committed to the backing store, then size of backing store increases.
+         * All the changes happen only on commit and not only put or remove.
+         */
+        @Test
+        @DisplayName("Put a key that does not already exist in the backing store")
+        void putNew() {
+            assertThat(state.readKeys()).isEmpty();
+            assertThat(state.modifiedKeys()).isEmpty();
+
+            state.put(C_KEY, CHERRY);
+            // Before commit, the size should be 1
+            assertEquals(2, state.size());
```

Comment:
The comment says size should be 1, but the test says the size is 2. But actually, it seems to me that the comment is right. How is the size 2?

Reply:
Since the size considered is backingStore size (And not modifications map which has size 1) , State's backing store already has `A -> Apple` and `B -> Banana` [here](https://github.com/hashgraph/hedera-services/blob/develop/hedera-node/hedera-app-spi/src/test/java/com/hedera/node/app/spi/state/WritableKVStateBaseTest.java#L51), when the state is created. So the size is 2. Modified the comment.